docs/src/clusters.md

```diff
@@ -44,12 +44,19 @@ solana config set --url https://api.devnet.solana.com
 $ solana-validator \
     --identity validator-keypair.json \
     --vote-account vote-account-keypair.json \
-    --known-validator dv1LfzJvDF7S1fBKpFgKoKXK5yoSosmkAdfbxBo1GqJ \
+    --known-validator dv1ZAGvdsz5hHLwWXsVnM94hWf1pjbKVau1QVkaMJ92 \
```

Comment:
--trusted-validator right? I use --know-validator in the master branch it does not like it

Reply:
Sorry -- I did not get the full picture. You are introducing a new parameters..

Reply:
Ah!  Yeah sorry I should've pointed that out earlier.  The t-word is a swear in this industry :confused: